code/game/machinery/OpTable.dm

```diff
@@ -30,6 +31,8 @@
 		computer = null
 	if(victim)
 		victim = null
+	injected_reagents.Cut()

```

Comment:
Yeah, you only need to worry about clearing variables/lists with object references, because those references will prevent those objects from GCing; even then, if they aren't being deleted at the exact same time, the references will probably get collected before it's an issue. A list of strings will just get collected when the object is deleted.